spring/src/test/java/org/axonframework/spring/serialization/avro/test1/ComplexObject.java

```diff
@@ -30,9 +30,6 @@
 public class ComplexObject extends org.apache.avro.specific.SpecificRecordBase
         implements org.apache.avro.specific.SpecificRecord {
 
-    private static final long serialVersionUID = -213207546923932305L;
```

Comment:
I believe this object was a generated result by Avro. Not sure if we should remove the `serialVersionUID` here. Did removal break any of the tests?